### PR TITLE
Conda Cheat Sheet

### PR DESCRIPTION
**Pull request by adityasiwan**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Conda} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

This PR adds conda cheat sheet.

###### Which issues (if any) does this fix?

None

###### People to notify (@mention interested parties)

@zekiel @tagawa 
---

Instant Answer Page: https://duck.co/ia/view/conda_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @adityasiwan

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/conda.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
+{
+    "id": "conda_cheat_sheet",
+    "name": "Conda",
+    "description": "Conda is an open source package management system and environment management system for installing multiple versions of software packages and their dependencies and switching easily between them.",
+
+    "metadata": {
+        "sourceName": "pydata.org",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/conda.json]: Lets change this to "PyData"

File: share/goodie/cheat_sheets/json/conda.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
+    "id": "conda_cheat_sheet",
+    "name": "Conda",
+    "description": "Conda is an open source package management system and environment management system for installing multiple versions of software packages and their dependencies and switching easily between them.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/conda.json]: remove the period at end of description

**Comment by adityasiwan** [comment]
@jophab Thanks for reviewing. Much appreciated. :+1:  
I have made the changes. :smile:

**Comment by adityasiwan** [comment]
@tagawa @zekiel If there are no further comments then this is good to go, imo. :smile:

**Comment by zekiel** [comment]
rgr lgtm! Thanks @adityasiwan!

**Pull request merged**